docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.9.1
+    environment:
+      CALYPSO_ENV: 'development'
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/calypso
+      - node_modules:/calypso/node_modules
```

Comment:
TIL.

> put this in some documentation

sounds great! maybe a quick note with a link to documentation on named bind points would be solid enough. we don't have to re document another project but pointing out this oddity would be awesome.